server/appengine/src/main/java/who/RefreshCaseStatsServlet.java

```diff
@@ -59,6 +59,10 @@ private static void updateData(
     if (timestamp > data.lastUpdated) {
       data.lastUpdated = timestamp;
     }
+    // Note that data.totalCases is a sum of dailyCases,
+    // while snapshot.totalCases is simply copied from the input data.
+    // The totalCases  at the most recent snapshot should match data.totalCases, but this
```

Comment:
1. double space after `totalCases`
1. newline before `data.totalCases` to make wrapping nicer on smaller screens

Reply:
haha, one of my editors has a bar at 80col and another at 100col. And i can't see the setting in android studio (though i see discussions of it online)